<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetUpWorkspace</class>
 <widget class="QDialog" name="SetUpWorkspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	border: 1px solid #959595;
}

QWidget {
	background: #FFFFFF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit_tabName"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_showCounter">
     <property name="text">
      <string>Show view counter on Tab</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_syncSelection">
     <property name="text">
      <string>Sync selection with fist Tab</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_syncZoom">
     <property name="text">
      <string>Sync zoom with first Tab</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
